src/test/java/org/fundacionjala/pivotal/hooks/Hooks.java

```diff
@@ -1,20 +1,39 @@
 package org.fundacionjala.pivotal.hooks;
 
+import com.jayway.restassured.response.Response;
 import cucumber.api.java.After;
-import org.fundacionjala.pivotal.core.api.APICommons;
+
 import org.fundacionjala.pivotal.core.api.APIRequestManager;
+import org.fundacionjala.pivotal.core.api.APICommons;
+import org.fundacionjala.pivotal.core.api.APIResponseName;
+
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)